### PR TITLE
Update didc release

### PR DESCRIPTION
**Pull request by sa-github-api**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Comment by bitdivine** [comment]
Diff with prod:

```
max@sinkpad:~/dfn/nns-dapp/branches/bot-didc-update (2:50:29)$ diff <(curl https://3r4gx-wqaaa-aaaaq-aaaia-cai.raw.icp0.io/v1/sns/list/latest/slow.json | jq .) <(curl https://otgyv-wyaaa-aaaak-qcgba-cai.raw.icp0.io/v1/sns/list/latest/slow.json | jq .)
    %%  TToottaall        %%  RReecceeiivveedd  %%  XXffeerrdd    AAvveerraaggee  SSppeeeedd      TTiimmee        TTiimmee          TTiimmee    CCuurrrreenntt

                                                                  DDllooaadd    UUppllooaadd      TToottaall      SSppeenntt        LLeefftt    SSppeeeedd

100 99014  100 99014    0     0   441k      0 --:--:-- --:--:-- --:--:--  441k         00  ----::----::----  ----::----::----  ----::----::----          00
100 99603  100 99603    0     0   325k      0 --:--:-- --:--:-- --:--:--  325k
26c26,27
<       "description": "A web3 social network governed by the people for the people"
---
>       "description": "A web3 social network governed by the people for the people",
>       "logo": "/v1/sns/root/u67kc-jyaaa-aaaaq-aabpq-cai/logo.png"
997,998c998,999
<       "sns_tokens_per_icp": 146.80606079101562,
<       "buyer_total_icp_e8s": 10898732954851,
---
>       "sns_tokens_per_icp": 150.26695251464844,
>       "buyer_total_icp_e8s": 10647716811851,
1000c1001
<       "direct_participant_count": 149,
---
>       "direct_participant_count": 147,
1038c1039,1040
<       "description": "Catalyze is a one-stop social-fi application for organising your Web3 experience"
---
>       "description": "Catalyze is a one-stop social-fi application for organising your Web3 experience",
>       "logo": "/v1/sns/root/uly3p-iqaaa-aaaaq-aabma-cai/logo.png"
1313c1315,1316
<       "description": "A web3 social network governed by the people for the people"
---
>       "description": "A web3 social network governed by the people for the people",
>       "logo": "/v1/sns/root/v2sfq-qyaaa-aaaaq-aabjq-cai/logo.png"
1588c1591,1592
<       "description": "BOOM DAO is an all-in-one web3 game platform and protocol running 100% on-chain on the Internet Computer"
---
>       "description": "BOOM DAO is an all-in-one web3 game platform and protocol running 100% on-chain on the Internet Computer",
>       "logo": "/v1/sns/root/xjngq-yaaaa-aaaaq-aabha-cai/logo.png"
1821,1822c1825,1826
<       "sns_tokens_per_icp": 1049.1436767578125,
<       "buyer_total_icp_e8s": 38126332938520,
---
>       "sns_tokens_per_icp": 1041.992431640625,
>       "buyer_total_icp_e8s": 38387992908520,
1824c1828
<       "direct_participant_count": 645,
---
>       "direct_participant_count": 649,
1861c1865,1866
<       "description": "Modclub is a decentralized crowdwork platform built on the IC that aims to support dApps by handing resource-intensive tasks such as moderation, user verification and data labeling."
---
>       "description": "Modclub is a decentralized crowdwork platform built on the IC that aims to support dApps by handing resource-intensive tasks such as moderation, user verification and data labeling.",
>       "logo": "/v1/sns/root/x4kx5-ziaaa-aaaaq-aabeq-cai/logo.png"
2067c2072
<     "icrc1_total_supply": 98000003564969980,
---
>     "icrc1_total_supply": 98000003564949980,
2151c2156,2157
<       "description": "Modclub is a decentralized crowdwork platform built on the IC that aims to support dApps by handing resource-intensive tasks such as moderation, user verification and data labeling."
---
>       "description": "Modclub is a decentralized crowdwork platform built on the IC that aims to support dApps by handing resource-intensive tasks such as moderation, user verification and data labeling.",
>       "logo": "/v1/sns/root/w7g63-nqaaa-aaaaq-aabca-cai/logo.png"
2402c2408,2409
<       "description": "The Next Generation of Decentralized Computing Network in AI Era"
---
>       "description": "The Next Generation of Decentralized Computing Network in AI Era",
>       "logo": "/v1/sns/root/55uwu-byaaa-aaaaq-aaa7q-cai/logo.png"
2641c2648,2649
<       "description": "Catalyze is a one-stop social-fi application for organising your Web3 experience"
---
>       "description": "Catalyze is a one-stop social-fi application for organising your Web3 experience",
>       "logo": "/v1/sns/root/5psbn-niaaa-aaaaq-aaa4q-cai/logo.png"
2925c2933,2934
<       "description": "The First Decentralized Meme Coin on IC"
---
>       "description": "The First Decentralized Meme Coin on IC",
>       "logo": "/v1/sns/root/4m6il-zqaaa-aaaaq-aaa2a-cai/logo.png"
3125c3134
<     "icrc1_total_supply": 988033227518357400,
---
>     "icrc1_total_supply": 988033227618357400,
3203c3212,3213
<       "description": "The First Decentralized Meme Coin on IC"
---
>       "description": "The First Decentralized Meme Coin on IC",
>       "logo": "/v1/sns/root/6kg2g-qaaaa-aaaaq-aaaxa-cai/logo.png"
```

File: rs/sns_aggregator/src/types/ic_sns_swap.patch [hunk 1]
```diff
@@ -171,18 +171,15 @@ index 1d3d0ca6f..fd164686d 100644
...
 -    Err(Err1),
-+pub enum Result_1 {
 +    Ok(Ok_1),
 +    Err(Err_1),
  }
  
  #[derive(CandidType, Deserialize, Serialize, Clone, Debug)]
- pub struct GetOpenTicketResponse {
--    pub result: Option<Result1>,
-+    pub result: Option<Result_1>,
```
Comment (bitdivine) [inline rs/sns_aggregator/src/types/ic_sns_swap.patch]: The main branch has `Result_1`.  The auto-generated code has `Result1`.  This commit adopts the name in the auto-generated code, as it is more natural.  As a side effect, the patch file becomes smaller.

File: rs/sns_aggregator/src/types/ic_sns_governance.patch [hunk 2]
```diff
@@ -1,5 +1,5 @@
 diff --git b/rs/sns_aggregator/src/types/ic_sns_governance.rs a/rs/sns_aggregator/src/types/ic_sns_governance.rs
-index 8a23bc900..d08b55a6d 100644
+index 8a23bc900..7fe1a341c 100644
 --- b/rs/sns_aggregator/src/types/ic_sns_governance.rs
 +++ a/rs/sns_aggregator/src/types/ic_sns_governance.rs
 @@ -8,8 +8,7 @@ use crate::types::{CandidType, Deserialize, EmptyRecord, Serialize};
```
Comment (bitdivine) [inline rs/sns_aggregator/src/types/ic_sns_governance.patch:5]: The main branch has `Result_1`.  The auto-generated code has `Result1`.  This commit adopts the name in the auto-generated code, as it is more natural.  As a side effect, the patch file becomes smaller and the code generation simpler.

**Review by lmuntaner — COMMENTED** [review]
There aren't any changes in the end, then?

**Comment by bitdivine** [comment]
@lmuntaner - There are, but the bot force pushed and that hid the manual changes needed to fix the bot's PR. :-D I'll add the manual changes again.

**Review by lmuntaner — APPROVED** [review]
LGTM

**Pull request merged**